apps/api/src/persistence/migrations/coscrad-migration.interface.ts

```diff
@@ -0,0 +1,11 @@
+import { ICoscradQueryRunner } from './coscrad-query-runner.interface';
+
+export interface ICoscradMigration {
+    sequenceNumber: number;
```

Comment:
How are we leveraging the sequenceNumber \ name? Are these redundant with any info that's on the decorator metadata?